packages/graphql/src/services/sessions.ts

```diff
@@ -1123,6 +1127,8 @@ export async function changeSessionSettings(
     where: { id },
     data: {
       isAudienceInteractionActive: isAudienceInteractionActive ?? undefined,
```

Comment:
As the isAudienceInteractionActive prop is not used anymore and will be dropped from the database, we should make sure that no DB service / frontend component still checks it 😅

Reply:
done ✅